client/src/legacy/SudoModePasswordField/SudoModePasswordFieldEntwine.js

```diff
@@ -0,0 +1,63 @@
+/* global window */
+import jQuery from 'jquery';
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { loadComponent } from 'lib/Injector';
+import qs from 'qs';
+
+jQuery.entwine('ss', ($) => {
+  $('.js-injector-boot .SudoModePasswordField').entwine({
+    Component: null,
+    ReactRoot: null,
+
+    onmatch() {
+      // onmatch will match both the field holder and the field
+      // we only want to run this on the field holder
+      if (this.is('input')) {
+        return;
+      }
+      this._super();
+      const cmsContent = this.closest('.cms-content').attr('id');
+      const context = (cmsContent)
+        ? { context: cmsContent }
+        : {};
+      const SudoModePasswordField = loadComponent('SudoModePasswordField', context);
+      this.setComponent(SudoModePasswordField);
+      this.refresh();
+    },
+
+    onunmatch() {
+      this._super();
+      const root = this.getReactRoot();
+      if (root) {
+        root.unmount();
+        this.setReactRoot(null);
+      }
+    },
+
+    /**
+     * This is used by the react component to reload the page after sudo mode is activated
```

Comment:
```suggestion
     * This is used by the react component to reload the page after sudo mode is activated.
     * This is here instead of inside the react component because in a pure react form the success handler
     * should result in re-rendering the react form with editable fields, instead of reloading the whole page.
```

Reply:
Updated